R2API.Rules/RulesPlugin.cs

```diff
@@ -0,0 +1,20 @@
+using BepInEx;
+using BepInEx.Logging;
+
+namespace R2API;
+
+[BepInPlugin(RuleCatalogExtras.PluginGUID, RuleCatalogExtras.PluginName, RuleCatalogExtras.PluginVersion)]
+public sealed class RulesPlugin : BaseUnityPlugin
+{
+    internal static new ManualLogSource Logger { get; set; }
+
+    private void Awake()
+    {
+        Logger = base.Logger;
+    }
+
+    private void OnDestroy()
```

Comment:
Can we use OnEnable / OnDisable pattern here?

Reply:
i mean, sure but then https://github.com/risk-of-thunder/R2API/pull/454#discussion_r1060210063 would need to return to make sure we dont re-add the custom rules if the module disables for whatever reason

Reply:
I think it will have to yes, the comment was just confusing since the proper way to fix it was just to unsub